src/github/githubApi.js

```diff
@@ -0,0 +1,16 @@
+const { isNil } = require("lodash");
+const graphql = require("graphql-request");
+
+module.exports = (key, query) => {
+  if (isNil(key)) {
+    throw new Error("Key not supplied");
```

Comment:
@K9Morphed Maybe just improve my poor messaging here. `GITHUB_API_KEY` not supplied?
And include some instructions in the README about how to add a key in Github and how to set the API key in the console before running the script?